tools/legal-review/engine/report-state

```diff
@@ -1,3 +1,3 @@
-FA401BD7EF3EA3C714DF95D0C1C72026C5E9AB5ACF48E3C88E120F90D270FF99
+68CB821CDFCC647A517E04B3ACC776E4F62F24B2D087816B165161517075BFE6
```

Comment:
The [license for JNA](https://github.com/enso-org/enso/blob/702d63f6a681e0e4ed8017db80787eb3a4545767/tools/legal-review/engine/net.java.dev.jna.jna-5.14.0/copyright-keep) is still part of `engine`.